cmd/plugin/rpaasv2/cmd/scale.go

```diff
@@ -5,55 +5,65 @@
 package cmd
 
 import (
-	"context"
 	"fmt"
 
-	"github.com/tsuru/rpaas-operator/pkg/rpaas/client"
-	"github.com/urfave/cli"
+	rpaasclient "github.com/tsuru/rpaas-operator/pkg/rpaas/client"
+	"github.com/urfave/cli/v2"
 )
 
-func initScaleFlags() []cli.Flag {
-	return []cli.Flag{
-		cli.StringFlag{
-			Name:     "service, s",
-			Usage:    "service name",
-			Required: true,
-		},
-		cli.StringFlag{
-			Name:     "instance, i",
-			Usage:    "instance name",
-			Required: true,
-		},
-		cli.IntFlag{
-			Name:     "quantity, q",
-			Usage:    "amount of units to scale to",
-			Required: true,
+func NewCmdScale() *cli.Command {
+	return &cli.Command{
+		Name:  "scale",
+		Usage: "Sets the number of replicas for an instance",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "service",
+				Aliases: []string{"tsuru-service", "s"},
+				Usage:   "the Tsuru service name",
+			},
+			&cli.StringFlag{
+				Name:     "instance",
+				Aliases:  []string{"tsuru-service-instance", "i"},
+				Usage:    "the reverse proxy instance name",
+				Required: true,
+			},
+			&cli.IntFlag{
+				Name:     "replicas",
+				Aliases:  []string{"quantity", "q"},
+				Usage:    "the new desired replicas number",
```

Comment:
I think this "new" is left here, what do you think of just "the desired number of replicas"?

Reply:
Sure.